### ISSUE TITLE
Namespace collision when fetching a contract from etherscan

### ISSUE THREAD

**Issue by Macarse** [eth-brownie/brownie#888]
### Environment information

* `brownie` Version: master
* `ganache-cli` Version: 6.12.1
* `solc` Version: 0.6.12
* Python Version: 3.x
* OS: osx

### Repro steps
Open a console with `brownie console --network mainnet-fork`
and run:

```
a = Contract.from_explorer("0x1D415aa39D647834786EB9B5a333A50e9935b796")
```

Output is
```
Fetching source of 0x1D415aa39D647834786EB9B5a333A50e9935b796 from api.etherscan.io...
  File "<console>", line 1, in <module>
  File "brownie/network/contract.py", line 909, in from_explorer
    _DeployedContractBase.__init__(self, address, owner)
  File "brownie/network/contract.py", line 455, in __init__
    self._check_and_set(abi["name"], fn)
  File "brownie/network/contract.py", line 474, in _check_and_set
    raise AttributeError(f"Namespace collision: '{self._name}.{name}'")
AttributeError: Namespace collision: 'AddressConfig._owner'
```

**Comment by iamdefinitelyahuman** [comment]
Ooof, that's a hard one.

Related to #718

**Comment by iamdefinitelyahuman** [comment]
Fixed in #961
